### PR TITLE
A few fixes against the futur website

### PR DESCRIPTION
**Pull request by trebmuh**
* one wrong link to the github wiki
* a few final point missing (for consistency)

I found these while translating the website into French.

### CODE REVIEW

File: templates/home.twig [hunk 1]
```diff
@@ -135,7 +135,7 @@
 
 <div>
 	<div class="page-header text-center">
-		<h2>Featuring</h2>
+		<h2>Featuring.</h2>
```
Comment (Umcaruje) [inline templates/home.twig]: I don't think a full stop makes sense here, the other changes are on point.

Comment (trebmuh) [reply templates/home.twig]: Suggestion: what about a ":" instead @Umcaruje ?

**Comment by Umcaruje** [comment]
A colon would work much better indeeed.

**Comment by trebmuh** [comment]
@Umcaruje : done.

**Comment by Umcaruje** [comment]
Thank you for the PR :+1:

**Pull request merged**